relay/abi.go

```diff
@@ -466,6 +469,40 @@ func handleLog(vLog *types.Log, w *wallet.Wallet, q *uploader.Queue, chainID uin
 
 	log.Println(res.RawLog)
 	log.Println(res.TxHash)
+
+	// callback on EVM chain
+	account, err := wEth.Derive(hdwallet.MustParseDerivationPath("m/44'/60'/0'/0/0"), false)
+	if err != nil {
+		log.Printf("Failed to derive account: %v", err)
+		return
+	}
+
+	// generate privkey and command
+	privKey, err := wEth.PrivateKeyHex(account)
+	if err != nil {
+		log.Printf("Failed to generate privkey: %v", err)
+		return
+	}
+
+	// 0 is a placeholder message id
+	cmdArgs := []string{"send", "--rpc-url", RPC, "--private-key", privKey, vLog.Address.Hex(), "finishMessage(string)", "0"}
```

Comment:
We can make the ID using this calculation from the contract:

```
uint height = block.number;
string memory s = string.concat(messageType, Strings.toHexString(uint160(sender), 20));
s = string.concat(s, Strings.toString(height));
```
where id is `s`

Reply:
Ah okay, if we're sure it won't change much

Reply:
note for myself: get sender and height from `vLog *types.Log`